innovation.scss

```diff
@@ -1670,6 +1693,29 @@
             background-position-x: -184px;
         }
     }
+    &.L {
+        top: 11px;
+        left: 191px;
+        width: 90px;
+        height: 90px;
+        background-size: 460px auto;
+        background-position-y: -330px;
+        &.color_0 {
```

Comment:
you should be able to use the functions here and get this in one line since the numbers are consistant 92px apart

Reply:
Agreed. But there's lots of other similar SCSS optimizations we can make later. So I'll defer this!